1-org/envs/shared/projects.tf

```diff
@@ -158,7 +158,7 @@ module "scc_notifications" {
   org_id                   = local.org_id
   billing_account          = local.billing_account
   folder_id                = google_folder.common.id
-  activate_apis            = ["logging.googleapis.com", "pubsub.googleapis.com", "securitycenter.googleapis.com", "billingbudgets.googleapis.com"]
+  activate_apis            = ["logging.googleapis.com", "pubsub.googleapis.com", "securitycenter.googleapis.com", "billingbudgets.googleapis.com", "cloudkms.googleapis.com"]
```

Comment:
For my understanding, are we adding new KMS features?
Or are the kms references just to demonstrate the code sample that the customer could use to enable CMEK encryption on the feed/function?

Reply:
Yes, I'm adding new KMS Features to use on:
- **Artifact Registry**
- Source code **Bucket**
- **Pub/Sub**

But the _cai-monitoring_ module itself accepts a null value for _encryption_key_.

Reply:
Today we have some KMS feature also on step 4-projects.
This KMS is to protect the resources that supports, like the ones that I listed above.

I added KMS feature in _cai-monitoring_ so the user can choose to use or not KMS protection.
But this module accepts a **null value** on **encryption_key** variable, and **true** on **enable_cmek** variable, that way it will not use KMS on the resources listed above.